charts/kargo/templates/NOTES.txt

```diff
@@ -107,6 +110,122 @@ Ready to get started?
    warning from your browser. You may safely disregard this.
 {{- end }}
 
+{{- end }}
+
+{{- if .Values.externalWebhooksServer.enabled }}
+
+{{- $https := false }}
+{{- $selfSignedCert := false }}
+{{- $url := "" }}
+
+{{- if .Values.externalWebhooksServer.ingress.enabled }}
+
+{{- $https = or .Values.externalWebhooksServer.ingress.tls.enabled .Values.externalWebhooksServer.tls.terminatedUpstream }}
+{{- $selfSignedCert = and .Values.externalWebhooksServer.ingress.tls.enabled .Values.externalWebhooksServer.ingress.tls.selfSignedCert (not .Values.externalWebhooksServer.tls.terminatedUpstream) }}
+{{- $url = include "kargo.externalWebhooksServer.baseURL" . }}
+
+⚙️  You've configured Kargo's external webhooks server to be accessible through
+   an Ingress controller.
+
+   If DNS and your Ingress controller are configured correctly, the external
+   webhooks server server is reachable at:
+
+{{- else if and .Values.api.enabled .Values.api.ingress.enabled }}
+
+{{- $https = or .Values.api.ingress.tls.enabled .Values.api.tls.terminatedUpstream }}
+{{- $selfSignedCert = and .Values.api.ingress.tls.enabled .Values.api.ingress.tls.selfSignedCert (not .Values.api.tls.terminatedUpstream) }}
+{{- $url = printf "%s/webhook" (include "kargo.api.baseURL" .) }}
+
+⚙️  You've configured Kargo's external webhooks server to be accessible through
+   an Ingress controller.
+
+   If DNS and your Ingress controller are configured correctly, the external
+   webhooks server server is reachable at:
+
```

Comment:
These lines carefully account for the scenario where one Ingress (that of the API server) is being used for both the API server _and_ the external webhooks server.

Reply:
Eh. I don't like how I did this. Refactoring.